ccswbs/gus
# Program template update

# Summary of changes
These are stepping-stone updates that will move majors higher on the page. Eventually this entire template can be revamped, but for now, this is a quick fix.

## Frontend

### Moving
- Move variants (e.g., Majors) below Program Overview

### Additions
   - Add four cards
   - Add Talk to a current Student

### Removals
  - Remove statistics
  - Remove variants (e.g, Majors) from accordion Program Information
  - Remove courses

### Commented out
  - Hide careers (rendering via General Text widget instead; leaving code in case template requires it later)

**Note:** I've left careers in the code (commented out) since they are still being rendered on the page as widgets. This is in case C&M still want to use the content type as their source in the New Year rather than a general text widget. (To be determined)

[x] My changes are accessible (at minimum WCAG 2.0 Level AA)
[x] My changes are responsive and appear as expected on mobile and desktop views.
[ ] After merging, I will update the [Content Hub documentation](https://uoguelphca.sharepoint.com/sites/UniversityContentHubInformationGroup) and ensure the Content Hub trainer understands how my changes will affect users.

## Backend
Before going live with this change, C&M is moving Program Benefits into the Program Overview field for all applicable programs.

# Test Plan

- View https://deploy-preview-182--ugconthub.netlify.app/programs/bachelor-of-applied-science-basc/
- View other programs at https://deploy-preview-182--ugconthub.netlify.app (under the Programs heading at the bottom)
- Confirm that pages operate as expected
- Linda will approve the layout
